### PR TITLE
Write Plog Reactions

### PR DESCRIPTION
**Pull request by jenzenho**
soln2cti.py was not writing out Plog reactions. Added in that functionality and it now spits out an error if it doesn't recognize a reaction type, just in case there are other reactions that are missed.

### CODE REVIEW

**Comment by xMestas** [comment]
Change seems good to me, I will let @kyleniemeyer take a look too.  Please update the comment at the top of the file to reflect the change, and thanks for contributing.

**Comment by kyleniemeyer** [comment]
Thanks @jenzenho!

@xMestas can you confirm that this new version works correctly? This reminds me that we should probably build tests for these functions.

**Comment by xMestas** [comment]
Regression tests still pass and everything seems to be working correctly.  Yea, the testing module definitely needs to be developed further.  I'll make that a priority along with gathering the comparison data.

File: pymars/soln2cti.py [hunk 1]
```diff
@@ -501,6 +514,33 @@ def build_species_string():
...
+            if equation_object.duplicate is True:
+                reaction_string = reaction_string + \
+                                '               options=\'duplicate\')\n\n'
+            else:
+                reaction_string = reaction_string[:-1]
+                reaction_string = reaction_string + ')\n\n'
+            f.write(reaction_string)
+        else:
+            print('Error: Unknown reaction type')
+            exit()
```
Comment (kyleniemeyer) [inline pymars/soln2cti.py]: This should raise an error or warning, rather than brute-force close. Maybe `raise ValueError('Unknown reaction type')`

**Comment by kyleniemeyer** [comment]
I think these changes look good. I just took a look at the file itself, and it needs quite a bit of work, but that isn't related to the PR, so I'm going to merge. Thanks @jenzenho for the contribution!

**Pull request merged**